### PR TITLE
Update sflow HLD according related code.

### PR DESCRIPTION
**Pull request by fredyu190011**
- What I did
Update the sflow HLD according the code in sonic-swss and sonic-utilities.

- Why I did it
The HLD is not match the code behavior.

- How I verified it
Below is related code:
https://github.com/Azure/sonic-utilities/blob/c7c01e4d30ccb88dd88645cd9390ed340e5ad119/config/main.py#L4243
https://github.com/Azure/sonic-swss/blob/321291aa7aca89a3ccef2bed36aff074eac1054f/cfgmgr/sflowmgr.h#L21
https://github.com/Azure/sonic-utilities/blob/c7c01e4d30ccb88dd88645cd9390ed340e5ad119/config/main.py#L4120
Related PR:
https://github.com/Azure/sonic-utilities/pull/1367

- Details if related

Signed-off-by: Fred Yu fred_yu@edge-core.com

### CODE REVIEW

**Comment by prsunny** [comment]
@dgsudharsan , @padmanarayana to review

File: doc/sflow/sflow_hld.md [hunk 1]
```diff
@@ -183,13 +183,14 @@ When sflow is disabled globally, sampling is stopped on all relevant interfaces
...
-  * 1-in-10,000  for a 10G link
-  * 1-in-40,000  for a 40G link
-  * 1-in-50,000  for a 50G link
-  * 1-in-100,000  for a 100G link
+  * 1-in-100 for a 1G link
+  * 1-in-1,000  for a 10G link
+  * 1-in-4,000  for a 40G link
+  * 1-in-5,000  for a 50G link
+  * 1-in-10,000  for a 100G link
+  * 1-in-40,000  for a 400G link
```
Comment (padmanarayana) [inline doc/sflow/sflow_hld.md:193]: Other changes look good. w.r.t the sample rate, discussed with @prsunny and @dgsudharsan. The HLD is good (i.e. the default sampling rate (of 1 in N packets sampled) should be (ifSpeed (in bits per sec) / 1e6).  This is based on recommendation from InMon (as described in https://blog.sflow.com/2013/06/large-flow-detection.html  : check the Link Speed/Sampling Rate table).  Can you please raise PR for https://github.com/Azure/sonic-swss/blob/321291aa7aca89a3ccef2bed36aff074eac1054f/cfgmgr/sflowmgr.h#L21 to change the SFLOW_SAMPLE_RATE_VALUE_* to be (ifSpeed / 1e6) ?

Comment (fredyu190011) [reply doc/sflow/sflow_hld.md:193]: Sure, I will raise another PR for this.

Comment (fredyu190011) [reply doc/sflow/sflow_hld.md]: Hi @padmanarayana , I have raised a PR: [sonic-swss#1623](https://github.com/Azure/sonic-swss/pull/1623).
I also restored the default sample rate to "ifSpeed / 1e6" in this PR.

**Review by padmanarayana — APPROVED** [review]

**Pull request merged**